app/controllers/albums_controller.rb

```diff
@@ -0,0 +1,64 @@
+class AlbumsController < ApplicationController
+  def get_album
+    id = params[:id]
```

Comment:
I like what you've done here! This `get_album` method would traditionally be a private method, which you could call directly as you have in your show method. The way you have it here, it looks like there should be a matching get_album view. To DRY your code up even more, you could refactor this controller to [use a filter to call the private `get_album` method](http://guides.rubyonrails.org/action_controller_overview.html#filters) before each controller action that needs to find an album by params[:id] and expose it to the view layer.